group-coordinator/src/main/resources/common/message/ConsumerGroupTargetAssignmentMemberValue.json

```diff
@@ -22,8 +22,10 @@
   "fields": [
     { "name": "TopicPartitions", "versions": "0+", "type": "[]TopicPartition",
       "about": "The assigned partitions.", "fields": [
-      { "name": "TopicId", "versions": "0+", "type": "uuid" },
-      { "name": "Partitions", "versions": "0+", "type": "[]int32" }
+      { "name": "TopicId", "versions": "0+", "type": "uuid",
+      "about": "The topic id."},
```

Comment:
The indentation does not look right here. `"about"` should be align with `"name"`, like it's done above.

This applies to many other files where you added `about` fields.